parser/shared/src/test/scala/hmda/parser/fi/lar/LarCsvParserSpec.scala

```diff
@@ -46,13 +46,25 @@ class LarCsvParserSpec extends PropSpec with PropertyChecks with MustMatchers wi
   property("Fail parsing when NA is not properly spelled") {
     forAll(larGen) { lar =>
       val na = lar.copy(rateSpread = "na")
-      LarCsvParser(na.toCSV).left.get mustBe LarParsingError(0, List("Rate Spread is not numeric or NA"))
+      LarCsvParser(na.toCSV).left.get mustBe LarParsingError(0, List("Rate Spread is not double or NA"))
```

Comment:
I think the word "double" is specific to programming. The message will probably be more useful to our users if it says "decimal" instead.